newswires/client/src/sharedTypes.ts

```diff
@@ -1,20 +1,47 @@
-export type WireData = {
-	id: number;
-	externalId: string;
-	ingestedAt: string;
-	content: Partial<{
-		uri: string;
-		usn: string;
-		version: string;
-		firstVersion: string; // date
-		versionCreated: string; // date
-		dateTimeSent: string; //date
-		headline: string;
-		subhead: string;
-		byline: string;
-		keywords: string;
-		usage: string;
-		location: string;
-		body_text: string;
-	}>;
-};
+import { z } from 'zod';
```

Comment:
I don't have strong feelings about validation libraries but my sense is that zod is the most widely used in the department at the moment